docs/install/dataplane-service-account.md

```diff
@@ -90,117 +108,83 @@ Plane:
-  the example configuration in `_example`.
-
-  Here, `default-cre-dataplane` refers to a Kubernetes Service Account bound to
-  the Google Cloud Service Account `cre-dataplane`. Remember to put this
-  Kubernetes Service Account name as the `spec.serviceAccountName` when you
-  create resources in the
+  After running the script, you will have a Kubernetes Service Account `example-ksa` in namespace `example`
+  which is bound to the Google Cloud Service Account `cre-dataplane` (you just created it in the last step).
+  Remember to put this Kubernetes Service Account name as the `spec.serviceAccountName`
+  when you create resources in the
   [example](https://github.com/google/knative-gcp/tree/master/docs/examples).
 
-  Kubernetes Service Account `default-cre-dataplane` doesn't need to exist in a
-  specific namespace. Once it is set in the ConfigMap `config-gcp-auth`， the
-  Control Plane will create it for you and configure the corresponding Workload
-  Identity relationship between the Kubernetes Service Account
-  `default-cre-dataplane` and the Google Cloud Service Account `cre-dataplane`
-  when you create resources using the Kubernetes Service Account
-  `default-cre-dataplane`.
-
-A `Condition` `WorkloadIdentityConfigured` will show up under resources'
-`Status`, indicating the Workload Identity configuration status.  
- **_Note:_** The Controller currently doesn’t perform any access control checks,
-as a result, any user who can create a resource can get access to the Google Cloud
-Service Account which grants the `iam.serviceAccountAdmin` permission to the Controller.
-As an example, if you followed the instructions above, then any user that can make
-a Knative-GCP source or Channel (e.g. `CloudAuditLogsSource`, `CloudPubSubSource`,
-etc.) can cause the Kubernetes Service Account `default-cre-dataplane` to be created.
-If they can also create Pods in that namespace, then they can make a Pod that uses
-the Google Service Account `cre-dataplane` credentials.
-
-### Option 2. Export Service Account Keys And Store Them as Kubernetes Secrets
-
-1. Download a new JSON private key for that Service Account. **Be sure not to
-   check this key into source control!**
-
-   ```shell
-   gcloud iam service-accounts keys create cre-dataplane.json \
-   --iam-account=cre-dataplane@$PROJECT_ID.iam.gserviceaccount.com
-   ```
+- **_Default scenario:_**
 
-1. Create a secret on the Kubernetes cluster with the downloaded key. Remember
-   to create the secret in the namespace your resources will reside. The example
-   below does so in the `default` namespace.
+  Instead of manually configuring Workload Identity namespace by namespace,
+  you can authorize the Controller to configure Workload Identity for you.
 
-   ```shell
-   kubectl --namespace default create secret generic google-cloud-key --from-file=key.json=cre-dataplane.json
-   ```
+  Apply [init_control_plane_gke.sh](../../hack/init_control_plane_gke.sh) without parameters:
```

Comment:
This should be init_control_planeXXX.sh or init_data_planeXXX.sh?

Reply:
data_plane, thank you for finding it.